packages/analytics/dashboard-renderer/sandbox/pages/RendererDemo.vue

```diff
@@ -43,6 +43,7 @@ const context: DashboardRendererContext = {
     type: 'relative',
     time_range: '24h',
   },
+  refreshInterval: 5000,
```

Comment:
i know this is just the demo page, but shouldn't we make it a little longer, like 20 seconds or so? :)

Reply:
It's just the sandbox, I wanna see it quick, not sit around for 20-30 seconds 😄